_sass/layout.scss

```diff
@@ -19,6 +19,21 @@
     }
 }
 
+.updated{
+    color: var(--color-secondary-dark);
+}
+
+.version{
+    color: var(--color-secondary-dark);
+}
+
+.version_no{
+    color: var(--color-secondary-dark);
+}
+
+.post-date{
+    color: var(--color-secondary-dark);
+}
 .anchor {
```

Comment:
Was there no previous styling for the `post-date` class ? If not , instead of making 4 different classes we could make it work with a single class as there isn't any other specific styling for them, let me know what you think about this

Reply:
Yeah I was actually confused between using different classes for them or using a single class for everyone.
I'll implement them using single class...

Reply:
@Avi-88 Have a look...

Reply:
Looks great, I know this is really me nitpicking now but could you please change the name of the class from `post-date` to something more general , like `post-details` or `post-info` , really sorry for this , thanks

Reply:
@Avi-88 You need not say sorry for that. I was the one who didn't realized it. I'll take care of it next time!

Reply:
There was already a class with `post-details` so renamed it to `post-info`.
Just drop a message if there's anything else that needs to be reworked or changed.